src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -143,36 +138,26 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
 
         #region Static initialization
 
-        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null, bool useJsonFallbackSerializer = false)
+        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null)
         {
             try
             {
                 RegisterBuiltInSerializers();
                 BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false));
                 RegisterSerializationProviders(serializationProviders);
                 AssemblyProcessor.Initialize();
-                fallbackSerializer = GetFallbackSerializer(useJsonFallbackSerializer);
+                fallbackSerializer = GetFallbackSerializer();
             }
             catch (ReflectionTypeLoadException ex)
             {
                 throw ex.Flatten();
             }
         }
 
-        internal static void Initialize(List<TypeInfo> serializationProviders, bool useJsonFallbackSerializer)
+        internal static void Initialize(List<TypeInfo> serializationProviders)
         {
             RegisterBuiltInSerializers();
-
-#if NETSTANDARD
```

Comment:
Maybe we should leave the warning here, and let `GetFallbackSerializer()` throw right after?